### DISCUSSION TITLE
Pause Button for recording Fieldborder

### DISCUSSION THREAD

**Discussion by Pecomica** [Courseplay/Courseplay_FS22#1341] [Ideas]
I am in favour of square and straight corners. 90 degree angles are the best for me. When using courseplay I can create a field using the mouse pointer on the worker menu map, but I struggle to get it completly square. Usually I end up 1 pixel above or below, and when using workers this tend to mess up a bit. Especially if you use a worker to create the field for you. 

I can record the field as I drive the edge, but I can't get a straight turn. It will always be a bit rounded so I was wondering if the following was planned or possible to implement.

1. Edit fields in the worker map view where we draw them using our mouse. This would allow me and others to get completly square corners.
2. Pause the recording when we drive the edge, so we can pause as we reach a corner. Orient our vehicle in the proper direction and resume the course, allow the corner to be square and nicely angled.
3. Another method could be to enter the coordinates of the corners, so we define the field that way. We just need to define corner 1, 2, 3, and 4 or how many we need to make the complete field. First and last coordinates are then combined with a line to close the field.

I do understand that this might not be a high priority for many, but it would help me a lot.
![Suggestion](https://user-images.githubusercontent.com/40060700/162114848-4ad5fbc8-f3ab-4851-a6b4-e7fcafc8284f.png)

**Comment by Tensuko** [comment]
Sry but this won't happen.
A Pause button might be an option.

**Comment by uf2-farmsim** [comment]
A pause button is a great idea.

**Comment by Tensuko** [comment]
Implemented in 7.2.1.0
